Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -1559,6 +1563,7 @@ - (void)URLSession:(NSURLSession *)session webSocketTask:(NSURLSessionWebSocketT
     configuration.URLCache = nil;
     configuration.allowsCellularAccess = existingConfiguration.allowsCellularAccess;
     configuration.connectionProxyDictionary = existingConfiguration.connectionProxyDictionary;
+    configuration.TLSMinimumSupportedProtocolVersion = existingConfiguration.TLSMinimumSupportedProtocolVersion;
```

Comment:
This covers modern WebKit, but not Legacy WebKit.

Reply:
I just added it to `ResourceHandleMac.mm`. Does that cover legacy WebKit?

Reply:
I think so, but actual experts should review. I now also wonder if WebSockets need any special handling.